### DISCUSSION TITLE
How to run Dagster with an up-to-date dbt project

### DISCUSSION THREAD

**Discussion by sburkulemb** [dagster-io/dagster#17759] [Q&A] [answered]
Hey guys got a question, I have hosted dagster on EC2, I want to keep the dbt repo updated(git pull) before the daily runs, whats the best practice to do that, will a simple bash script be the best option, or should I try out some other tool?

**Comment by rexledesma** [comment, marked as answer]
We don't recommend that you pull the dbt repo at run time. Instead, the dbt project should be packaged with your Dagster code at build time.

See our docs for guidance: https://docs.dagster.io/integrations/dbt/reference#deploying-a-dagster-project-with-a-dbt-project
